microsoft-edge/devtools-guide-chromium/storage/cache.md

```diff
@@ -24,7 +24,7 @@ ms.date: 05/04/2021
 
 This guide shows you how to use DevTools to inspect [Cache](https://developer.mozilla.org/docs/Web/API/Cache) data.
 
-To inspect [HTTP cache](https://developer.mozilla.org/docs/Web/HTTP/Caching) data, in the **Network** tool, use the **Size** column of the **Network Log**.  See [Log network activity](../network/index.md#log-network-activity) in _Inspect network activity_.
+To inspect [HTTP cache](https://developer.mozilla.org/docs/Web/HTTP/Caching) data, in the **Network** tool, use the **Size** column of the **Network Log**.  See [Log network activity](../network/index.md#log-network-activity) in _Inspect network activity_.  See [Quota Useage](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/progressive-web-apps/#quota-usage)
```

Comment:
Same comments here, the link here seems a bit arbitrary. There should be a clearer explanation about why we're linking to this particular page, and I don't think it belongs in this paragraph.

Also, the link is incorrect: it shouldn't be absolute but should instead be a relative path. Also, when you need to use an absolute URL (when linking to a 3rd-party site for example), don't include the `/en-us` part. This is usually automatically added by the site depending on the preferred language of the user.

So, in this case, I would do the following: at the end of the "View cache data" section, add a paragraph with something like: "To learn more about cache data storage quota, see ..."

Reply:
fixed